src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -18,11 +18,6 @@
     <SuppressGenerateILCompilerExplicitPackageReferenceWarning>true</SuppressGenerateILCompilerExplicitPackageReferenceWarning>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(UseNativeAotForComponents)' == 'true'">
```

Comment:
We may want to keep this around and condition it on `MicrosoftDotNetILCompilerVersion` being set. It is likely that we will need to take newer ILC that what's in SDK at some point in future again to take a critical fix temporarily.

Reply:
(We are on similar plan with Roslyn: We use Roslyn from SDK in steady state, but we may use a newer one temporarily.)